docs/lib/datastore/getting-started.md

```diff
@@ -1,52 +1,136 @@
 ---
 title: Getting started
-description: Amplify DataStore provides a programming model for leveraging shared and distributed data without writing additional code for offline and online scenarios, which makes working with distributed, cross-user data just as simple as working with local-only data.
+description: The Amplify DataStore category enables developers to write applications whose data lives in the mobile device for instant/offline-first access while synchronization with a cloud backend is handled automatically.
 ---
 
-## Datastore with Amplify
-
-Amplify DataStore provides a programming model for leveraging shared and distributed data without writing additional code for offline and online scenarios, which makes working with distributed, cross-user data just as simple as working with local-only data.
+The Amplify DataStore category enables developers to write applications whose data lives in the mobile device for instant/offline-first access while synchronization with a cloud backend powered by [AWS AppSync](https://aws.amazon.com/appsync/) is handled automatically, which makes working with distributed, cross-user data just as simple as working with local-only data.
```

Comment:
This is a bit of a run-on sentence.  I feel like we should lead in with the problem that developers face when... the wording needs to be updated, but something like:
Handling data in a modern mobile application is difficult due to data constantly changing coupled with poor network conditions.  Meanwhile, users increasingly demand "snappy" applications which has lead to techniques like optimistic UI which assume network calls will eventually succeed.  The Amplify DataStore category enables developers to adopt a programming model where data lives on device so that it is instantly available even when customers have a poor internet connection.  Furthermore, ... etc.. etc..

I'm also fine to just leave it as you have it right now, and update it at a later time.

Reply:
I'm open to suggestions but I have a strong opinion that the current content doesn't tell me what DataStore is. I can only understand the current paragraph if I open that page knowing that I'm looking at a offline/cloud-sync database solution.

When I look at the other categories I can easily understand what they do right away.

Is there a way to describe DataStore in a way someone who only used CoreData or text files to persist data would understand?

Reply:
By popular demand and lack of a better option I reverted to the original introduction paragraph.